rainforest/rainforest.go

```diff
@@ -0,0 +1,120 @@
+// Package rainforest is a golang client for the Rainforest QA API
+package rainforest
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+)
+
+const (
+	currentBaseURL  = "https://app.rainforestqa.com/api/1/"
```

Comment:
In the Ruby version, this is also configurable with a ENV var - are we keeping that?